apps/sms/lib/bridge/client.js

```diff
@@ -606,10 +606,11 @@ var debug = {
 
 /**
  * Default response timeout.
+ *
```

Comment:
Mmm, maybe, I see some JSDoc changes, I can pick the most new version if you want :)